test/test_jit_fuser_te.py

```diff
@@ -1210,7 +1213,8 @@ def rand(dtype, device="cuda"):
                 continue
             t = torch.jit.trace(fn, (x,))
             self.assertEqual(ref, t(x))
-            self.assertAllFused(t.graph_for(x))
+            if dtype in supported_dtypes:
```

Comment:
What do we test when we run scripted but not fused graphs (when the dtype is unsupported)?

Reply:
Basically this is just testing that we don't generate something that throws (which is what would happen before this diff).  But it's kind of ridiculous to test that for every single unary op.  Let me split the unsupported dtypes into a separate tiny test.